test-requirements.txt

```diff
@@ -1 +1 @@
-odoo-test-helper
```

Comment:
Why the deletion?

Reply:
I've fixed it. Maybe it was mistakenly deleted.